locales/pl.json

```diff
@@ -1,5 +1,5 @@
 {
-  "nav": {
+  "Nawigacja": {
```

Comment:
please don't change this values, since it's necessary to navigate through the object sturcture

Reply:
Thank you for the hint. Will adjust and send a new version.